### DISCUSSION TITLE
Request for a complete service

### DISCUSSION THREAD

**Discussion by mhfaisalbd** [OpenCodeFoundation/eSchool#360] [General]
On the master branch, I found [](https://github.com/OpenCodeFoundation/eSchool/tree/master/src/Services/Enrolling) as service, but there is not any fully implemented domain model which contains all properties. Can any one update a fully workable service?

**Comment by mahedee** [comment]
You can follow Enrolling service. It's fully completed service. If you think, any upgradation required, create an issue against that service or code.

**Comment by ratanparai** [comment]
I have planned to work on #171 which is planned for [Milestone 10](https://github.com/OpenCodeFoundation/eSchool/milestone/10)
